src/tests/ftest/cart/corpc/cart_corpc_five_node.py

```diff
@@ -17,7 +17,8 @@ class CartCoRpcFiveNodeTest(CartTest):
     def test_cart_corpc(self):
         """Test CaRT CoRPC.
 
-        :avocado: tags=all,cart,pr,daily_regression,corpc,five_node
+        :avocado: tags=all,cart,pr,daily_regression
```

Comment:
The first `tags=` line should just be "buckets" that the test runs in.  I.e.

```suggestion
        :avocado: tags=all,pr,daily_regression
```

Then the `cart` tag would go onto the next `tags=` line.

This applies to others in other files but I won't comment on each individually.

Reply:
2894719c0168038fc96d4e6446fcc242bb20a45d